src/main/java/org/broadinstitute/hellbender/tools/copynumber/arguments/GermlineContigPloidyModelArgumentCollection.java

```diff
@@ -0,0 +1,74 @@
+package org.broadinstitute.hellbender.tools.copynumber.arguments;
+
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.hellbender.tools.copynumber.DetermineGermlineContigPloidy;
+import org.broadinstitute.hellbender.utils.param.ParamUtils;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * @author Mehrtash Babadi &lt;mehrtash@broadinstitute.org&gt;
+ */
+public final class GermlineContigPloidyModelArgumentCollection implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    @Argument(
+            doc = "Contig-level mean bias standard deviation.  If a single sample is provided, " +
+                    "this input will be ignored.",
+            fullName = "mean-bias-standard-deviation",
```

Comment:
Perhaps extract these to `public static final` in this class.

Reply:
Done.